app/main.py

```diff
@@ -1,5 +1,72 @@
+from __future__ import annotations
+
+
 class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+        self,
+        name: str,
+        weight: int,
+        coords: list = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = coords if coords is not None else [0, 0]
```

Comment:
You could make this line shorter in this way
```suggestion
        self.coords = coords if coords else [0, 0]
```
or using `or`
```suggestion
        self.coords = coords or [0, 0]
```